packages/react-composites/src/composites/CallComposite/components/LobbyTile.tsx

```diff
@@ -48,15 +48,8 @@ export const LobbyTile = (props: LobbyTileProps): JSX.Element => {
             aria-atomic
           >
             <Stack.Item className={mergeStyles(titleStyle(palette, isVideoReady))}>
-              {props.overlayProps.overlayIcon}
+              <Spinner size={SpinnerSize.large} label={props.overlayProps.title}></Spinner>
             </Stack.Item>
-            <Text
```

Comment:
is there an ask to remove this text along with the icon?

Reply:
Please also make sure this still looks correct when making a PSTN and 1:n call 😊

Reply:
We have decided to not use spinner label and use existing text as of now.